app/test/test_datastore.py

```diff
@@ -0,0 +1,383 @@
+        tableName = 'Sessions'
+        table = self.create_session_table()
+        d = DataStore('dynamodb', tableName, 'PatientId', 'SessionId')
+        info = {DATASTORE_COLUMN_SESSION_ID : 's-1',
+                DATASTORE_COLUMN_PATIENT_ID: 'p-1',
+                DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_ID: 'h-1',
+                DATASTORE_COLUMN_SESSION_NAME: 'SessionName',
+                DATASTORE_COLUMN_COMPREHEND_S3_PATH: 'ComprehendS3Path',
+                DATASTORE_COLUMN_TRANSCRIBE_S3_PATH: 'TranscribeS3Path',
+                DATASTORE_COLUMN_TIMESTAMP_START: 1969,
+                DATASTORE_COLUMN_TIMESTAMP_END: 2030}
+        response = d.save(info, 'us-west-2')
+        response = table.scan()
+        items = response['Items'] if 'Items' in response else {}
+        item = items[0] if items else None
+        self.assertTrue('ComprehendS3Path' in item[DATASTORE_COLUMN_COMPREHEND_S3_PATH])
+        self.assertEqual('p-1', item[DATASTORE_COLUMN_PATIENT_ID])
+
+        # test save none value
+        info[DATASTORE_COLUMN_PATIENT_ID] = 'p-2'
+        info[DATASTORE_COLUMN_COMPREHEND_S3_PATH] = ''
+        response = d.save(info, 'us-west-2')
+        self.assertEqual('BAD', response['status'])
+        self.assertEqual('ComprehendS3Path should not be empty.', response['error'])
+        response = table.scan()
+        items = response['Items'] if 'Items' in response else {}
+        for item in items:
+            self.assertTrue('p-2' not in item[DATASTORE_COLUMN_PATIENT_ID])
+
+        # test save empty value
+        info[DATASTORE_COLUMN_PATIENT_ID] = 'p-3'
+        info[DATASTORE_COLUMN_COMPREHEND_S3_PATH] = None
+        response = d.save(info, 'us-west-2')
+        self.assertEqual('BAD', response['status'])
+        self.assertEqual('ComprehendS3Path should not be empty.', response['error'])
+        response = table.scan()
+        items = response['Items'] if 'Items' in response else {}
+        for item in items:
+            self.assertTrue('p-3' not in item[DATASTORE_COLUMN_PATIENT_ID])
+
+        # tableName,healthCareProfessionalId,healthCareProfessionalName,healthCareProfessionalIdVal,healthCareProfessionalNameVal='HealthCareProfessionals','HealthCareProfessionalId','HealthCareProfessionalName','h-3e3477c37d674ecc98e3cdf5487ee07b','World'
+        # table = self.create_health_care_professional_table()
+        # d = DataStore('dynamodb', tableName, healthCareProfessionalId)
+        # d.save({healthCareProfessionalId: healthCareProfessionalIdVal, healthCareProfessionalName: healthCareProfessionalNameVal},'us-west-2')
+        # response = table.scan()
+        # items = response['Items'] if 'Items' in response else {}
+        # item = items[0] if items else None
+        # self.assertTrue(healthCareProfessionalIdVal in item[healthCareProfessionalId])
+        # self.assertEquals(item[healthCareProfessionalName], healthCareProfessionalNameVal)
+
+    @mock_dynamodb2
+    def test_list_items_sessions(self):
```

Comment:
Ideally, Each test should check one case.
The test name should ideally be closer to the lambda name. Could you refactor this to test_list_sessions instead of test_list_items_Sessions?
Could you split this into more tests in smaller units where each unit checks only one condition?


For example ,
test_list_sessions_no_session,
test_list_sessions_one_session,
test_list_session_no_patient_information,
test_list_Session_no_healthcare_professional_information

Reply:
Done.